app/Models/Chat/UserChannel.php

```diff
@@ -95,7 +95,15 @@ public static function presenceForUser(User $user)
             })
             ->join('channels', 'channels.channel_id', '=', 'user_channels.channel_id')
             ->where('channels.type', '=', 'PM')
-            ->with(['userScoped.friends', 'userScoped.blocks'])
+            ->with([
+                // only fetch data related to $user, to be used by ChatStart privilege check
```

Comment:
I think it should mention it's where `$user` is the target of the friend/block since it's not getting any of the users `$user` has friended or blocked?